src/Components/IndividualArticle.jsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getIndividualArticle } from "../api";
+import formatDate from "../utils/utils";
```

Comment:
this import was breaking the code when I ran it so please make sure this is fixed before merging to main